test/stages/stage0.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.0.0"
```

Comment:
should this be >= 1.3.0?

Reply:
No optional arguments were used in this example hence this change 1instead this should have been as it is i.e. `>=0.13.0` 
But I have modified this to use latest version i.e. `>=1.3.0`